lib/project/ext/ui_view_controller.rb

```diff
@@ -79,14 +79,16 @@ def viewDidLoad
     end
   end
 
-  def pm_handles_did_load?
+  def pm_handles_delegates?
     self.is_a?(ProMotion::ViewController) || self.is_a?(ProMotion::TableScreen)
   end
 
   def view_will_appear(animated)
   end
   def viewWillAppear(animated)
-    self.view_will_appear(animated)
+    unless pm_handles_delegates?
+      self.view_will_appear(animated)
```

Comment:
similar to viewDidLoad - this is called in PM, so we don't want to call it a second time around.